docs/recipes/decouplingReducers.md

```diff
@@ -0,0 +1,141 @@
+# Decoupling reducers
+
+Rematch and more generally Redux, encourage to keep a flat state to solve several [issues](https://redux.js.org/recipes/structuring-reducers/normalizing-state-shape).
+Each model should be considered as a separate entity and should only be updated by its reducer, this is why Rematch doesn't allow nested reducers.
+
```

Comment:
Nested reducers can be used in Rematch, but are not recommended. I think that is an important distinction.

Some users have set up functions, similar to combineReducers, that generate their nested reducers. I've also seen people use redux.reducers with combineReducers.

Reply:
I would say, this is not possible out of the box: https://github.com/rematch/rematch/issues/282#issuecomment-372644100